lesson9/src/methods/Main.java

```diff
@@ -0,0 +1,55 @@
+package methods;
+
+import java.util.*;
+
+public class Main {
+    public int countOccurance(List <?> strings, Object word) {
+        var count = 0;
```

Comment:
incorrect input data, please be attentive to the task description, it needs tcountOccurance needs to get a List of strings and random strings ( in your case I see a list of Jeneriks and object)